tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -5375,6 +5375,10 @@ public function findEntryFromPropfindResponse(
 		if ($entryNameToSearch === null) {
 			return $results;
 		}
+
+		//print all response of search request if match not found
+		print_r($multistatusResults);
```

Comment:
If this is needed for CI. how about just doing an assertion step here. sth like:
```suggestion
		Assert::assertFail("Expected to be something :) but got something else. \n Mulistatus results is here:" . print_r(multistatusResults, true))
```

Reply:
the printed response contains a response to the search response.It saves our time and gives some idea why tests are failing